### ISSUE TITLE
Request for scripts of TableFact dataset

### ISSUE THREAD

**Issue by fmh1art** [yunjiazhang/ReAcTable#5]
Appreciate for nice work!

I can run your script on WikiTQ dataset. However, I find that there may be a small error in `notebooks\ReAcTable-MajorityVote-TabFact.ipynb`. The content seems to be the code for WikiTQ dataset. Can you update a new script?

**Comment by BlacksheepJane** [comment]
Hi, I noticed that you closed this issue. I'm facing the same problem now. Did you manage to resolve it?

**Comment by fmh1art** [comment]
> Hi, I noticed that you closed this issue. I'm facing the same problem now. Did you manage to resolve it?

Yes, I reproduced the result on TabFact with the guidance of the authors.

**Comment by BlacksheepJane** [comment]
> > Hi, I noticed that you closed this issue. I'm facing the same problem now. Did you manage to resolve it?
> 
> Yes, I reproduced the result on TabFact with the guidance of the authors.

That's great to hear! Do you still have the original script you used? If so, would you mind sharing it? It would really help me reproduce the results as well.

Thanks in advance!
